### ISSUE TITLE
Knex: Timeout acquiring a connection. The pool is probably full.

### ISSUE THREAD

**Issue by himanshuoodles** [tgriesser/knex#2141]
Hi @tgriesser ,
I am facing "Unhandled rejection TimeoutError: Knex: Timeout acquiring a connection. The pool is probably full. Are you missing a .transacting(trx) call?" issue while making insert query to postgres database from 2 same application servers. How did you solve this problem ?
Please suggest some solution.

**Comment by elhigu** [comment]
There is probably something wrong how you are using knex. Its impossible to say without seeing the failing code. Please add minimal test code, which can be used to reproduce the problem. 

Also this issue tracker is mainly for development, not for user support, so stackoverflow would be better medium for this.

**Comment by atiertant** [comment]
@elhigu got the same problem using knex@0.12.9 on oracledb dialect via the stack [offshore](https://github.com/Atlantis-Software/offshore) / [offshore-sql](https://github.com/Atlantis-Software/offshore-sql)

**Comment by Glavin001** [comment]
@atiertant What did you do to fix this problem with knex@0.12.9 and OracleDB?

**Comment by atiertant** [comment]
@Glavin001 you should have a look at this: 
```
$ sqlplus / as sysdba
select resource_name, current_utilization, max_utilization, limit_value from v$resource_limit where resource_name in ('processes', 'sessions', 'transactions');
```
